utility_analysis_new/tests/dp_engine_test.py

```diff
@@ -106,29 +112,39 @@ def test_utility_analysis_params(self):
         params_with_custom_combiners.custom_combiners = sum
```

Comment:
It's a little bit difficult to understand what behavior is being tested by this test. Would you mind renaming the test and separating the arrange, act and assert blocks with blank lines or comments ?
Are we testing that "test_invalid_utility_analysis_params_throws_exception"?